### PR TITLE
Clean up styling and the frontend codebase

### PR DESCRIPTION
**Pull request by Lev-Stambler**
- Removed tailwind
- Added very primitive styles as well as a color scheme w/ CSS variables
- Moved all base components into a seperate NX library
- Decoupled state from Near components
- Factored out Near chain specific code into the Near library. There is one ugly thing here, please see #54
- Changed layouts to use CSS grid

A lot of files changed, but mostly just moving small things around.

### CODE REVIEW

File: apps/frontend/src/components/SendForm.svelte [hunk 1]
```diff
@@ -1,27 +1,36 @@
...
   export let onCancel: () => void | undefined;
   export let chain: Chain;
 
+  let chainSendFormPart;
   const { open } = getContext('modal');
 
-  const handleSubmit = async (v: any) => {
-    console.log(`submit: ${v}`);
+  // TODO: clean up imports, see https://github.com/bafnetwork/baf-wallet-v2/issues/54
+  const ChainSendFormPart = (chain: Chain) => () =>
```
Comment (Sladuca) [inline apps/frontend/src/components/SendForm.svelte:19]: oof yeah I feel like we definitely want a better system for this, but I think it's ok for now - we have an issue and in the long run I think this is somewhat OK if we're careful / clean about not letting this jank leak elsewhere.

File: apps/frontend/src/components/SendForm.svelte [hunk 2]
```diff
@@ -34,15 +43,19 @@
...
-    <div class="flex flex-row justify-around pt-3">
-      {#if onCancel !== undefined}
-        <Button onClick={onCancel}>Cancel</Button>
-      {/if}
-      <Button type="submit">Submit</Button>
-    </div>
-  {:else}
-    Please enter a valid chain through which to send funds
-  {/if}
+  <!-- TODO: bind createTX somehow!! -->
```
Comment (Sladuca) [inline apps/frontend/src/components/SendForm.svelte:46]: Add issue for this

File: apps/frontend/src/components/SendForm.svelte [hunk 3]
```diff
@@ -34,15 +43,19 @@
...
+  <Lazy
+    component={ChainSendFormPart(chain)}
+    chainInterface={$ChainStores[chain]}
+    edPK={$SiteKeyStore.edPK}
+    secpPK={$SiteKeyStore.secpPK}
+    bind:selfBind={chainSendFormPart}
+  />
+  <!-- <SendNearFormPart bind:createTX /> -->
+  <div class="flex flex-row justify-around pt-3">
```
Comment (Sladuca) [inline apps/frontend/src/components/SendForm.svelte:55]: we're not using tailwind anymore right?

File: apps/frontend/src/pages/Account.svelte [hunk 4]
```diff
@@ -40,85 +40,38 @@
...
+    <p>
+      It looks like you have not setup your account on any of the Blockchains
+      which we support.
+    </p>
+    <p>
+      Please see initialize your <a href="/settings" use:link>account's page</a>
+    </p>
+  {:else}
+    <h1>Account</h1>
+    <!-- <div>
```
Comment (Sladuca) [inline apps/frontend/src/pages/Account.svelte:57]: should commented code be removed or are you planning on doing something with this?

**Review by Sladuca — CHANGES_REQUESTED** [review]
This is awesome! Just 1-2 tiny things and some questions

File: apps/frontend/src/components/SendForm.svelte (hunk 2 above)
Comment (Lev-Stambler) [reply apps/frontend/src/components/SendForm.svelte:46]: Woops, this is done, I need to remove comment

File: apps/frontend/src/components/SendForm.svelte (hunk 1 above)
Comment (Lev-Stambler) [reply apps/frontend/src/components/SendForm.svelte:19]: Yeah agreed

File: apps/frontend/src/components/SendForm.svelte (hunk 3 above)
Comment (Lev-Stambler) [reply apps/frontend/src/components/SendForm.svelte:55]: Yeah, will remove

File: apps/frontend/src/pages/Account.svelte (hunk 4 above)
Comment (Lev-Stambler) [reply apps/frontend/src/pages/Account.svelte:57]: Woops again, I think that was there before, I'll j remoe for now

**Review by Sladuca — APPROVED** [review]
boats are cool 🚢

**Pull request merged**